src/main/java/com/automation/school/ad_main/Collection/MainRetailor.java

```diff
@@ -0,0 +1,97 @@
+package com.automation.school.ad_main.Collection;
+
+import java.util.*;
+
+public class MainRetailor {
+    public static void main(String[] args) {
+        //Start
+        System.out.println("Hello to  create new oder: 1) enter user name 2) indecat type 3) enter item name" );
+        // New user registration
+        User user1 = new User();
+        Scanner myNewUserObj = new Scanner(System.in);  // Create a Scanner object
```

Comment:
it's a scanner object has nothing to do with user/address, so variable name should not confuse reader with user object

Reply:
Was replaced with myScannerForUser. is it correct name in this case?